packages/node/src/indexer/runtime/runtime.service.spec.ts

```diff
@@ -41,21 +42,22 @@ const getApiService = (): ApiService =>
             // Treat 0 as 1 for parent hash of block 1
             const blockNum = Math.max(parseInt(blockHash), 1);
 
-            const { id } = specVersions.find(
+            const specVersion = specVersions.find(
               (v) => v.start <= blockNum && (!v.end || v.end >= blockNum),
             );
+            assert(specVersion, `Spec version not found for block ${blockNum}`);
 
             return Promise.resolve({
               specVersion: {
-                toNumber: () => parseInt(id),
+                toNumber: () => parseInt(specVersion.id),
```

Comment:
Not related to the PR but this can lead to issues if not provided. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt#radix

```suggestion
                toNumber: () => parseInt(specVersion.id, 10),
```